ifc/templates/party/fraternityPartyList.html

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Yes, that's the ideal place to call `render_template`. Take a look at the flask docs